lamindb/curators/core.py

```diff
@@ -452,7 +484,13 @@ def __init__(
                     required = feature.uid not in optional_feature_uids
                 else:
                     required = False
-                if feature.dtype in {"int", "float", "num"}:
+                # series.dtype is "object" if the column has lists types, e.g. [["a", "b"], ["a"], ["b"]]
+                are_lists = (
+                    False
+                    if feature.name not in self._dataset.keys()
+                    else isinstance(self._dataset[feature.name][0], (list, np.ndarray))
```

Comment:
This isn't a robust way to check that the entire series has elements that are iterable. Can't we do anything better? 🤔